src/__configuration__/themes/schedule.ts

```diff
@@ -18,16 +18,16 @@ import { IndependenceDaySchedule } from './independence-day';
  * Add more holiday themes here.
  *
  * Current schedules:
- * * Feb 11 - Feb 18: Spring Festival (changes each year)
+ * * Feb 1 - Feb 7 2022: Spring Festival (changes each year)  ** Jan 21 - Jan 27 2023 | Feb 9 - Feb 15 2024 **
  * * Mar 7 - Mar 9: Women's Day
  * * Mar 16 - Mar 18: St. Patrick's Day
  * * Apr 20 - Apr 24: Earth Day
  * * May 3 - May 5: May the Fourth Day
  * * July 2 - July 7: Independence Day
- * * Oct 14 - Nov 2: Halloween
```

Comment:
Do we want to shorten this one a little? We made this looooong last year bc it was our first theme in the new docit, but maybe this year we don't need to prepare for the trick-or-treat for 2 weeks...

Reply:
I agree, I think a week for halloween is plenty